templates/types/extractor/fastapi/README-template.md

```diff
@@ -51,12 +51,10 @@ To retrieve a response with low confidence since the question is not related to
 
 You can start editing the API endpoint by modifying [`extractor.py`](./app/api/routers/extractor.py). The endpoints auto-update as you save the file.
 
-Open [http://localhost:8000/docs](http://localhost:8000/docs) with your browser to see the Swagger UI of the API.
-
 The API allows CORS for all origins to simplify development. You can change this behavior by setting the `ENVIRONMENT` environment variable to `prod`:
 
 ```
-ENVIRONMENT=prod python main.py
+poetry run reflex run --env prod
```

Comment:
in app/app.py (previously is main.py) we are using ENVIRONMENT variable (default is dev if it's not specified). But from now, I think set default as `prod` would be better.

Reply:
yes agree